src/platform/plugins/shared/workflows_management/server/workflows_management/routes/post_test_workflow.ts

```diff
@@ -29,10 +30,28 @@ export function registerPostTestWorkflowRoute({ router, api, logger, spaces }: R
     async (context, request, response) => {
       try {
         const spaceId = spaces.getSpaceId(request);
+        const esClient = (await context.core).elasticsearch.client.asCurrentUser;
+
+        let processedInputs = request.body.inputs;
+        try {
+          processedInputs = await preprocessAlertInputs(
+            request.body.inputs,
+            spaceId,
+            esClient,
+            logger,
+            'test'
+          );
+        } catch (preprocessError) {
+          logger.debug(
```

Comment:
some comment as above

Reply:
Updated - added check if this is expected to be an alert before trying to preprocess, and if yes and failed - logging as error